homeassistant/components/tradfri/light.py

```diff
@@ -100,6 +100,15 @@ def is_on(self) -> bool:
             return False
         return cast(bool, self._device_data.state)
 
+    @property
+    def color_mode(self) -> ColorMode | None:
+        """Return the color mode of the light."""
+        if self._fixed_color_mode:
+            return self._fixed_color_mode
+        if self.hs_color:
+            return ColorMode.HS
+        return ColorMode.COLOR_TEMP
```

Comment:
I'm not sure if this is the correct way to distinguish between hs and color_temp, @ggravlingen do you think it's OK?

Reply:
Looks OK to me.